database/postgresql.go

```diff
@@ -38,10 +38,20 @@ type PostgreSQL struct {
 	tables        []string
 	excludeTables []string
 	password      string
+	compress      string
+	format        string
 	args          string
 	_dumpFilePath string
 }
 
+var (
+	PostgreSQLCompressionExt = map[string]string{
+		"gzip": "gz",
+		"lz4":  "lz4",
+		"zstd": "zst",
+	}
+)
+
```

Comment:
Remove this `format` check, allows any values. 

We do not need to do to mush limit, user can follow the PostgreSQL document to do.

Reply:
@huacnlee In general I agree, but it is beneficial to set the correct file extension. This saves hassle while restoring from the backup when the file is named accordingly. What do you think. How do you propose to handle file extension if we do not have this filter? We can ofc add another config param with file extension if you prefer :-)

Reply:
@huacnlee I would appreciate if you could share your opinion. I need this compression param for backing up large (>100GB) databases. If there is no hope to merge and release it soon, I will need to base it on the fork. Thanks!

Reply:
Remove this check, then I will going to merge this PR. I want Gobackup to not too many logic.

Reply:
Actually, no this PR change, we also can use `args` option for this case, any arguments that `pg_dump` supports can assign by `args` option.

Reply:
I see, only use `args` we can't change the output filename.